app/design/adminhtml/Magento/backend/Magento_Sales/web/css/source/module/_order.less

```diff
@@ -306,4 +306,14 @@
     }
 }
 
+.giftmessage-whole-order-container{
```

Comment:
Styles for the gift message module should be in the corresponded folder and hardcode margin for the field it's not a good idea because it can break page layout for other devices with different resolutions.